frontend/src/components/Events/EventsContainer.js

```diff
@@ -22,13 +26,31 @@ const SCContainer = styled(Container)({
   textAlign: "center",
 });
 
+
 function EventsContainer(props) {
+
+  const viewAll = () => {
+    axios.get(`http://localhost:3001/events`).then( res => {
```

Comment:
Same comment as above, server URL should exist in env variable, and the endpoint `"/events" should be appended to the end of it.